Python/stackrefs.c

```diff
@@ -216,4 +216,12 @@ PyStackRef_IsNullOrInt(_PyStackRef ref)
     return PyStackRef_IsNull(ref) || PyStackRef_IsTaggedInt(ref);
 }
 
+_PyStackRef
+PyStackRef_IncrementTaggedIntNoOverflow(_PyStackRef ref)
+{
+    assert(ref.index != (uintptr_t)-1); // Overflow
```

Comment:
Do here same as in https://github.com/python/cpython/pull/132555/commits/d9dc597a713c3fa72557520796b65b94ecea7189 ?